--- a/etc/uci-defaults/10_migrate-shadow	2020-09-28 05:05:55.000000000 -0400
+++ b/etc/uci-defaults/10_migrate-shadow	2022-12-25 17:17:00.681549715 -0500
@@ -6,7 +6,6 @@
 if [ -n "${ppwd#[\!x]}" ] && [ -z "${spwd#[\!x]}" ]; then
 	logger -t migrate-shadow "Moving root password hash into shadow database"
 	sed -i -e "s:^root\:[^\:]*\::root\:x\::"     /etc/passwd
-	sed -i -e "s:^root\:[^\:]*\::root\:$ppwd\::" /etc/shadow
 fi
 
 exit 0
